main.tf

```diff
@@ -95,20 +97,23 @@ module "google_mysql_db" {
   read_replica_name_suffix         = local.read_replica_name_suffix
   read_replicas = [
     for array_index in range(var.read_replica_count) : {
-      name = array_index
-      tier = var.instance_size_read_replica
-      zone = local.zone_read_replica
+      name                  = array_index
+      tier                  = var.instance_size_read_replica
+      zone                  = local.zone_read_replica
+      availability_type     = null
+      disk_type             = "PD_SSD"
+      disk_autoresize       = true
```

Comment:
Why removing `var.disk_auto_resize_read_replica` ?

Reply:
added it back